jvm-libs/linea/clients/linea-l1-contract-client/src/main/kotlin/linea/contract/Web3JContractAsyncHelper.kt

```diff
@@ -292,7 +292,7 @@ class Web3JContractAsyncHelper(
     blobs: List<ByteArray>,
     gasPriceCaps: GasPriceCaps?
   ): SafeFuture<String> {
-    require(blobs.size in 0..6) { "Blobs size=${blobs.size} must be between 0 and 6." }
+    require(blobs.size in 0..9) { "Blobs size=${blobs.size} must be between 0 and 9." }
```

Comment:
As you said yourself on the call, we could parametrize this or factor this check out somewhere and reuse

Reply:
This might be a leftover for a while, but I wonder why not check for `blobs.size in 1..9` here instead, I dun see any case where we'll be given empty blobs when calling this function

Reply:
Good point - adjusted the check